tools/c7n_mailer/c7n_mailer/sqs_message_processor.py

```diff
@@ -20,12 +20,11 @@
 import yaml
 import zlib
 
-from address import get_user
+from ldap_lookup import LdapLookup
 from boto3 import Session
 from email.mime.text import MIMEText
 from email.utils import parseaddr
-from functools import partial
```

Comment:
no need to do partials anymore, we just have a long lived LdapLookup object that is an instance variable for each mailer run.